### ISSUE TITLE
FEATURE: recommendations refresh tool

### ISSUE THREAD

**Issue by baob** [swapmyvote/swapmyvote#764]
Recommendations are going to keep changing during the election. Both for live data and developer sanity I think we need a tool that will destroy and rebuild from scratch the TV recommendations table.

**Comment by aspiers** [comment]
Agreed, although I believe @joshr said they shouldn't change too much.

**Comment by baob** [comment]
> I believe @joshr said they shouldn't change too much.

Although there are 3 more integrations besides MVTFWD to consider, there's a PR for one and @tdg has suggested two more.

**Comment by aspiers** [comment]
Worst case: `heroku run rails c` and then manually load the relevant seed library, instantiate the class and call `#load`.
